device_doctor/tool/build.sh

```diff
@@ -6,7 +6,7 @@
 # Fetches corresponding dart sdk from CIPD for different platforms, builds
 # an executable binary of device_doctor to `build` folder.
 #
-# This currently supports linux, mac and windows.
+# This currently supports linux and mac.
```

Comment:
If we implement this logic in the recipe we don't need to branch out scripts for different platforms. How are we getting the dart sdk?

Reply:
Dart sdk is defined in the ensure_file, obtained via cipd.